src/js/tracks/track-list.js

```diff
@@ -72,7 +72,8 @@ class TrackList extends EventTarget {
        */
       this.trigger({
         track,
-        type: 'addtrack'
+        type: 'addtrack',
+        target: this
```

Comment:
I think the target here is intended to be the tracklist that the event is triggered on and not document.

Reply:
makes sense